src/main/java/com/patchworkmc/Patchwork.java

```diff
@@ -265,6 +267,10 @@ private void transformMod(ForgeModJar forgeModJar) throws IOException, URISyntax
 
 		LOGGER.trace("fabric.mod.json: " + json);
 
+		// Write annotation data
+		Path annotationJsonPath = fs.getPath("/annotations.json");
```

Comment:
Annotation data should only be written if it isn't empty, IMO. The comment about putting the annotation data path into the fabric.mod.json applies here as well.